frontend/packages/dev-console/src/components/helm/HelmReleasesDetailsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { match as RMatch } from 'react-router';
+import * as _ from 'lodash';
+import { Firehose } from '@console/internal/components/utils';
+import { NamespaceBar } from '@console/internal/components/namespace';
+import HelmReleaseDetailsPage from './HelmReleaseDetailsPage';
+
+export interface HelmReleasesDetailsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+
+const HelmReleasesDetailsPage: React.FC<HelmReleasesDetailsPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  const helmReleaseName = match.url.substring(_.lastIndexOf(match.url, '/') + 1);
+  return (
+    <>
+      <NamespaceBar />
+      <Firehose
+        resources={[
+          {
+            namespace,
+            kind: 'Secret',
+            prop: 'secret',
+            isList: true,
+            selector: { name: `${helmReleaseName}` },
```

Comment:
Is this just the name of the resource? If so then use the name prop. And why is  `isList` true?

Reply:
using the name prop.
The overview page doesn't load if the isList is not true